scripts/js/basic_usage.js

```diff
@@ -57,7 +57,15 @@ async function main(){
   log('>>>>>>');
 
   // WebSockets
-  web3 = new Web3(process.env.INFURA_WSS);
+  web3 = new Web3(
+    process.env.INFURA_WSS,
+    {
+      clientConfig: {
+        maxReceivedFrameSize: 100000000,
```

Comment:
One message can fit into one or more(if message is fragmented) data frames as https://datatracker.ietf.org/doc/html/rfc6455#section-5.6 , 

The default values are `maxReceivedFrameSize` 1MiB and `maxReceivedMessageSize` 8MiB
https://github.com/theturtle32/WebSocket-Node/blob/a2cd3065167668a9685db0d5f9c4083e8a1839f0/docs/WebSocketClient.md#client-config-options 

Both `maxReceivedFrameSize` and `maxReceivedMessageSize` are made here around 95.3 MiB, which will hinder fragmentation and multiplexing https://datatracker.ietf.org/doc/html/rfc6455#section-5.4 in case of huge message.

Plus It will not make ws connection fail safe.

Reply:
As these are just the tests so be fine. But I agree with @jdevcs we should be using realistic values here.